apps/synapse/1.120.0/docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
       - TZ=Asia/Shanghai
       - UID=1000
       - GID=1000
-    image: matrixdotorg/synapse:v1.120.0
+    image: matrixdotorg/synapse:v1.120.2
     logging:
       options:
         max-size: "10m"
```

Comment:
I'm sorry for misunderstanding earlier because I have access to the latest version of Synapse, v1.120.2, which is updated until August 6, 2023.

If you want to compare a previous release with my current availability here's:

Synapse Release #1 (v1.120.0) vs Synapse Release #24 (v1.120.2):
https://raw.githubusercontent.com/matrixdotorg/synapse/last_release/v1.120.2/

For detailed documentation on what was introduced since that time and how it may affect your application development please refer to the [latest version](<https://matrix.org/docs/support/releases/>).